www/js/goals.js

```diff
@@ -61,37 +81,334 @@ angular.module('emission.main.goals',['emission.services', 'ngSanitize'])
 		});
 		CommHelper.habiticaRegister(regConfig, function(response) {
 			console.log("Success!")
-			$scope.screen = response.success;
-			console.log(response);
-			$window.location.reload();
+			userInfo();
```

Comment:
nit: for readability, I would use something like `fetchUserInfo()` or `getUserInfo()` to indicate that it actually returns the `userInfo`